tangoObjects.py

```diff
@@ -317,23 +323,23 @@ def set(self, id, obj):
         self.dict[str(id)] = obj
 
     def get(self, id):
-        if str(id) in self.dict.keys():
+        if str(id) in self.dict:
             return self.dict[str(id)]
         else:
             return None
 
     def keys(self):
-        return self.dict.keys()
+        return list(self.dict.keys())
```

Comment:
doesn't it return a list here? 

Wouldn't calling list again on keys() be redundant in the other lines I have highlighted above? Such as in
`if vmName not in list(self.machines.keys()):`